openmc/stats/multivariate.py

```diff
@@ -375,8 +375,8 @@ class SphericalIndependent(Spatial):
     r"""Spatial distribution represented in spherical coordinates.
 
     This distribution allows one to specify coordinates whose :math:`r`,
-    :math:`\theta`, and :math:`\phi` components are sampled independently from
-    one another and centered on the coordinates (x0, y0, z0).
+    :math:`\cos_theta`, and :math:`\phi` components are sampled independently
```

Comment:
Since we are still effectively sampling theta independently of r and phi, I think it makes sense to leave this as is
```suggestion
    :math:`\theta`, and :math:`\phi` components are sampled independently
```

Reply:
Looks like this also needs to be updated